### PR TITLE
bug:merge conflict

### PR DESCRIPTION
**Pull request by rubymaharjan**
conflict due to same class names

### CODE REVIEW

**Comment by Xkid0525** [comment]
use Font Awesome instead of images for brand icons like Facebook, Twitter, LinkedIn

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+}
+.profile-container{
+    background-color: white;
+    height: 315px;
+    width: 70%;
+    margin-left: 15%;
+    margin-right: 15%;
+    margin-top: -120px;
+}
+.profile-container .column {
```
Comment (Xkid0525) [inline styles.css]: Don't prefer to use this kind of class definition, use it only when you have other tags with the same class names
try making class names as unique as possible,

Plus the column definition is a little wonky,
when overflow is hidden .

File: portfolio.html [hunk 2]
```diff
@@ -19,7 +21,7 @@ <h2>Ruby Maharjan</h2>
                 <div class="grid-right">
                     <ul class="icons">
                         <li><a href="www.facebook.com"><img src="facebookIcon.png"></a></li>
```
Comment (khadkarajesh) [inline portfolio.html]: @rubymaharjan use vector rather than pngs

File: portfolio.html [hunk 3]
```diff
@@ -77,6 +79,41 @@ <h1>I'm Ruby Maharjan</h1>
                     </div>
                 </div>
             </div>
+            <div class="container-skill">
+                <div class="heading">
+                    <h2>Skills</h2>
+                </div>
+              <div class="box row1">
```
Comment (khadkarajesh) [inline portfolio.html]: @rubymaharjan use the proper name in a way that your code tells the story

**Comment by khadkarajesh** [comment]
@rubymaharjan did you resolve the comments that are suggested  above.

**Pull request merged**